rs/bitcoin/consensus/src/payload_builder.rs

```diff
@@ -343,17 +348,17 @@ impl BatchPayloadBuilder for BitcoinPayloadBuilder {
         let since = Instant::now();
 
         let delivered_ids = parse::parse_past_payload_ids(past_payloads, &self.log);
-        let payload = match self.get_self_validating_payload_impl(context, delivered_ids, max_size)
-        {
-            Ok(payload) => payload,
-            Err(e) => {
-                log!(self.log, e.log_level(), "{}", e);
-                self.metrics
-                    .observe_build_duration(e.to_label_value(), since);
-
-                return vec![];
-            }
-        };
+        let payload =
+            match self.get_self_validating_payload_impl(context, delivered_ids, max_size, 0) {
```

Comment:
Why do we always pass priority `0` here? Is this `impl` only used in tests?

Reply:
I see makes sense. In that case I think we should create a ticket to delete this code; we can always add it back in case we need it.